package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "war-server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "husky": {
```

Comment:
instead of putting this in `package.json` which I try to avoid, put this in `.huskyrc.js` as a `module.exports`

Reply:
while we're adding that .js file. add a `pre-push` to run jest. That way we can't push to the repo without tests passing! 💯

Reply:
once you do this jest will be block you from pushing, so you will need to fix your failing test.